### PR TITLE
Add Ansible-lint to the build pipeline

### PR DESCRIPTION
**Pull request by inponomarev**
This one adds Ansible-lint to the build pipeline.

Both YAMLLint and Ansible-lint are prerequisites for the `Molecule` automated testing system.

* Added ansible-linting to Travis build
* Fixed simple and unambigious issues, such as using `shell` instead of `command` and missing task names.
* For all the other cases added `skip_ansible_lint` tag and added TODO comments.
* Removed the duplicated yamllint.yml file and increased the line length threshold from 80 to 120.

Most unresolved Anslible-lint issues relate to the places in code that are just 'ansibilized' shell scripts, without guaranteed idempotency and usage of respecting Ansible modules.

Deeper refactoring is possible, but, as any refactoring, it requires the automated tests. 

The further steps could be adding `molecule` testing to some of the roles.

### CODE REVIEW

**Comment by inponomarev** [comment]
Ooops, sorry @karianna, I missed the change requests! I will try to find some time and rebase this PR soon!

**Comment by karianna** [comment]
Conflicts.

File: ansible/playbooks/AdoptOpenJDK_Services_Playbooks/ubuntu-jckservices.yml [hunk 1]
```diff
@@ -54,86 +59,110 @@
...
           with_items:
             - kadmin.local -q "addprinc -pw `pwgen -1` admin/admin@ADOPTOPENJDK_NET"
             - kadmin.local -q "addprinc -pw user1 user1/jckservices.adoptopenjdk.net@ADOPTOPENJDK_NET"
             - kadmin.local -q "addprinc -pw user2 user2/jckservices.adoptopenjdk.net@ADOPTOPENJDK_NET"
             - kadmin.local -q getprincs | egrep '^admin/admin@|^user1/|^user2/' > krb5.jckusers.txt; if test $(wc -l < krb5.jckusers.txt) -ne 3; then echo Wrong number of users - expected 3:; cat krb5.jckusers.txt; rm krb5.jckusers.txt; exit 1; fi
           args:
             creates: krb5.jckusers.txt
-        - service:
+          tags:
+            - skip_ansible_lint
```
[the remark this answers is not in the document]
Comment (inponomarev) [reply ansible/playbooks/AdoptOpenJDK_Services_Playbooks/ubuntu-jckservices.yml]: added comments with explanation

**Comment by inponomarev** [comment]
@karianna I rebased the branch and fixed/muted the warnings introduced since June

**Pull request merged**